docs/running-notebooks/listing-active-sessions/index.md

```diff
@@ -0,0 +1,15 @@
+# Listing active sessions
+
+You can list the active Jupyter notebooks sessions from the REANA
+command-line client [reana-client](https://pypi.org/project/reana-client/)
+in the same way you list workflows and additionally passing the
```

Comment:
Perhaps shorter: ...reana-client using the `list` command with the `--sessions` option: